src/lib.rs

```diff
@@ -150,6 +157,12 @@ pub struct HotShot<CONSENSUS: ConsensusType, TYPES: NodeType, I: NodeImplementat
     /// The hotstuff implementation
     hotstuff: Arc<RwLock<Consensus<TYPES, I::Leaf>>>,
 
+    // TODO: use associated traits to pair these maps with respective exchanges
+    // e.g. per exchange
+    // struct ChannelMaps {
+    //    proposal_channel:Arc<RwLock<SendToTasks<TYPES, I>>>,
+    //    votes_channel<:Arc<RwLock<SendToTasks<TYPES, I>>>,
+    // }
```

Comment:
Should we open an issue for this?

Reply:
Yes! I just created an [issue](https://github.com/EspressoSystems/HotShot/issues/1193) and will add a link here.